las_geoh5/import_las.py

```diff
@@ -241,7 +249,14 @@ def add_data(
         if existing_data and isinstance(existing_data, Entity):
             kwargs["entity_type"] = existing_data.entity_type
 
-        drillhole.add_data({name: kwargs}, property_group=property_group)
+        try:
+            drillhole.add_data({name: kwargs}, property_group=property_group)
+        except ValueError as err:
+            msg = (
+                f"ValueError raised trying to add data '{name}' to "

```

Comment:
This is so that a failed import on one file doesn't terminate the whole import process.  One case this covers is a las file containing data belonging to property group a (that already exsists on a particular drillhole) with a different depth sampling.
![image](https://github.com/MiraGeoscience/las-geoh5/assets/81254271/1288a406-f5e9-400b-bde7-85d58b369b63)
Compared to:
![image](https://github.com/MiraGeoscience/las-geoh5/assets/81254271/5ad40c51-5392-4d5c-bbec-35951060e555)

Reply:
Although it would technically catch any old ValueError.  I could update geoh5py to raise a custom error and catch that instead.. would be more safe.